api/src/services/db-doc.js

```diff
@@ -45,35 +56,42 @@ module.exports = {
   // this filters audited endpoints, so valid requests are allowed to pass through to the next route
   filterOfflineRequest: (userCtx, params, method, query, body) => {
     const isAttachment = params.attachmentId;
+    let stored;
+    let requested;
 
     return Promise
       .all([
         getStoredDoc(params, method, query, isAttachment),
         getRequestDoc(method, body, isAttachment),
-        authorization.getAuthorizationContext(userCtx)
       ])
-      .then(([ storedDoc, requestDoc, authorizationContext ]) => {
-        if (!storedDoc && !requestDoc) {
+      .then(([ storedDoc, requestDoc ]) => {
+        stored = getDocCtx(storedDoc);
+        requested = getDocCtx(requestDoc);
+
+        return authorization.getScopedAuthorizationContext(userCtx, [ stored, requested ]);
+      })
+      .then(authorizationContext => {
+        if (!stored && !requested) {
```

Comment:
Why `getScopedAuthorizationContext` when we know we're going to return false here? Can this be moved up to the previous block to return before the additional call?